checkstyle/import-control.xml

```diff
@@ -276,10 +280,7 @@
         <allow pkg="kafka.utils" />
         <allow pkg="org.apache.zookeeper" />
         <allow pkg="org.apache.zookeeper" />
```

Comment:
This duplication has been addressed in [this PR](https://github.com/apache/kafka/pull/8130).

Since this setting contols only direct imports only, it does not bring `log4j` transitively.